kubernetes/bin/update-deployment-travis.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+if [ "$TRAVIS_REPO_SLUG" != "loklak/loklak_server" ]; then
+    echo "Skipping image update for repo $TRAVIS_REPO_SLUG"
+    exit 0
+fi
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+    echo "Skipping image update for pull request"
+    exit 0
+fi
+
+if [ "$TRAVIS_BRANCH" != "master" ] && [ "$TRAVIS_BRANCH" != "development" ]; then
+    echo "Skipping image update for branch $TRAVIS_BRANCH"
+    exit 0
+fi
+
```

Comment:
Could these if conditions for this shell code and push_docker.sh be coded at one place?
like adding these cinditions in `.preload_travis.sh` (like `.preload.sh` in `./bin`) and added in each files.